src/components/Map/GoogleMapComp.tsx

```diff
@@ -1,28 +1,25 @@
-import { useEffect, useState } from 'react';
+import getCoordsFromAddress, { Coord, House } from '@/utils/services/getCoordsFromAddress';
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
 import { GoogleMap, Marker, withGoogleMap, withScriptjs } from 'react-google-maps';
-
-import getCoordsFromAddress from '/src/utils/services/getCoordsFromAddress';
-
 import styles from './GoogleMapComponent.module.scss';
+import CSS from 'csstype';
 
-export default function GoogleMapComp({ houses, style }) {
-  function Map() {
-    const [housesCoords, setHousesCoords] = useState([]);
 
+export default function GoogleMapComp({ houses, style }:{houses:House,style:CSS.Properties}) {
```

Comment:
It could be a good idea to declare an interface or type with `{houses:House,style:CSS.Properties}` above the function to make it easier to read.